### PR TITLE
SPARKTA-110 Improve Cassandra Output

### PR DESCRIPTION
**Pull request by danielcsant**
#### Description
Possibility to set the cluster name and all the cassandra-spark options (https://github.com/datastax/spark-cassandra-connector/blob/master/doc/reference.md#read-tuning-parameters)

#### Reviewer
@gserranojc   

#### Test
All passed.

#### Coverage
No changes

#### Scalastyle
No info

### CODE REVIEW

**Comment by gserranojc** [comment]
OK!!

**Comment by Gschiavon** [comment]
OK!

File: plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala [hunk 1]
```diff
@@ -123,9 +127,27 @@ object CassandraOutput {
...
       ("spark.cassandra.connection.port", connectionPort)
     )
   }
+
+  private def getSparkCassandraProperties(configuration: Map[String, JSerializable]): Seq[(String, String)] = {
+    configuration.get("sparkProperties") match {
+      case Some(properties) =>
+        val conObj = configuration.getMapFromJsoneyString("sparkProperties")
+        conObj.map(c => {
+          val key = c.get("sparkPropertyKey").get
```
Comment (sgomezg) [inline plugins/output-cassandra/src/main/scala/com/stratio/sparkta/plugin/output/cassandra/CassandraOutput.scala]: Please use c("sparktaPropertyKey") if you are assuming that this property is mandatory. Maybe you could use a better name for "c", like propKeyPair for example.

**Comment by danielcsant** [comment]
Thanks!

**Comment by sgomezg** [comment]
LGTM :)

**Pull request merged**